modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/login-button/index.html

```diff
@@ -0,0 +1,3 @@
+<div class="evp-login-button">
+	<a class="btn btn-login" href="/c/portal/login" type="button">Sign in</a>
```

Comment:
I never see this `type` applied to a <a tag before, can you remove it?
```suggestion
	<a class="btn btn-login" href="/c/portal/login">Sign in</a>
```

Reply:
Resolve !